source/includes/_01-your-first-backend-in-10-minutes.md

```diff
@@ -0,0 +1,204 @@
+# Building your first serverless backend in 10 minutes 
+
+Building your serverless application backend have never been this easy. In this section, we will go through all the necessary steps to have your backend up and running in a few minutes.
+
+If you follow the steps below, you will be able to create the application backend for a blog.
+
+The steps to follow will be:
+- [Create project](#1-create-the-project)
+- [First command](#2-first-command)
+- [First event](#3-first-event)
+- [First entity](#4-first-entity)
+- [First read model](#5-first-read-model)
+- [Deployment](#6-deployment)
+- [Testing](#7-testing)
+    - [Creating posts](#71-creating-posts)
+    - [Retrieving all posts](#72-retrieving-all-posts)
+    - [Retrieving specific post](#73-retrieving-specific-post)
+- [Removing stack](#8-removing-stack)
+- [Further improvements](#9-further-improvements)
+- [Conclusion](#10-conclusion)
+
+## 1. Create project
+First of all, we need to create a base Booster project
+
+`boost new:project <project-name>`
+
+You will need to answer a few questions in order to configure the project. One of these steps will require that a provider is selected, we recommend selecting AWS. 
```

Comment:
I'd rephrase this as

```suggestion
You will need to answer a few questions in order to configure the project. The last step asks you about a _provider package_, for this tutorial, select AWS.
```

Otherwise, it won't work for the user